packaging/wheel/install.sh

```diff
@@ -57,28 +59,3 @@ fi
 
 cd $startdir
 rm -r xrootdbuild
-
-# TODO: Remove all of the below and build a wheel using PyPA tools
-
-# convert the egg-info into a proper dist-info
-egginfo_path=$(ls $1/xrootd-*.egg-info)
-core="${egginfo_path%.*}"
-core="${egginfo_path#$1/}"
-sufix="${core#xrootd-*.*.*_-}"
-core="${core%_-*}"
-if [[ "$core" == "$sufix" ]]
-then
-    distinfo_path="${egginfo_path%.*}.dist-info"
-else
-    distinfo_path="$1/$core-$sufix"
-fi
-echo $distinfo_path >> /tmp/out.txt
-mkdir $distinfo_path
-mv $egginfo_path $distinfo_path/METADATA
-echo -e "Wheel-Version: 1.0\nGenerator: bdist_wheel (0.35.1)\nRoot-Is-Purelib: true\nTag: py2-none-any\nTag: py3-none-any" > $distinfo_path/WHEEL
-touch $distinfo_path/RECORD
-distinfo_name=${distinfo_path#"$1"}
-find $1/pyxrootd/      -type f -exec sha256sum {} \; | awk '{printf$2 ",sha256=" $1 "," ; system("stat --printf=\"%s\" "$2) ; print '\n'; }' >> $distinfo_path/RECORD
-find $1/$distinfo_name -type f -exec sha256sum {} \; | awk '{printf$2 ",sha256=" $1 "," ; system("stat --printf=\"%s\" "$2) ; print '\n'; }' >> $distinfo_path/RECORD
-find $1/XRootD/        -type f -exec sha256sum {} \; | awk '{printf$2 ",sha256=" $1 "," ; system("stat --printf=\"%s\" "$2) ; print '\n'; }' >> $distinfo_path/RECORD
-find $1/pyxrootd/ -type l | awk '{print$1 ",,"}' >> $distinfo_path/RECORD
```

Comment:
Unless there is some specific reason to _need_ to rename things to act like a wheel instead of an egg for Pythons that have old pips then this should get removed to simplify things and to avoid files not existing causing errors during the build.

If you have Python 3.3+ and a modern pip then `pip` can handle the build and install of a wheel from the sdist. Otherwise (that is you're on Python 3.2 (which is very very EOL so please don't) or using a pip that is quite old) you'll get a totally fine egg and can be used or uninstalled normally, so there's no clear motivation (to me) to go through attempting to renamed its `egg-info` dir to `dist-info`.

Reply:
@matthewfeickert i believe that the lowest baseline is centos 7 that have python 3.6.8 and pip 9.0.3 (setuptools 39.2.0 wheel 0.31.1) so anything lower than this can go

Reply:
Great! That's what I assumed. CentOS 7 is going to hold people at Python 3.6 for a long time, but I don't know of any real world cases where people are still using any Python 3.X version older than Python 3.6. :+1: